Updating INL-HPC documentation

aeslaughter opened the pull request:
(refs #14767)

<!--
INCLUDE THE FOLLOWING IN THE PR DESCRIPTION
- Explain relevant design information for your change.
- Follow the [Coding Standards](http://mooseframework.org/wiki/CodeStandards/).
- Submit or improve [Test Cases](http://mooseframework.org/wiki/MooseTraining/testing/).
- Reference a specific issue, place "refs #<issue>" or "closes #<issue>" (e.g., #closes #1234).
-->

aeslaughter commented:
I still need to add the PBS example, but I just wanted to see if it builds

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/497732/) on 06c6bd5 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/15046/site)

This comment will be updated on new commits.

aeslaughter committed:
hit format simple diffusion

This files is used in MooseDocs tests so the associated gold files were also changed

(refs #14767)

aeslaughter commented:
@permcody This is ready for a review. The new INL-HPC landing page is here: https://mooseframework.inl.gov/docs/PRs/15046/site/help/inl/index.html

This will need to get merged to next with the broken apps docs. Bison and Mastodon will need a change and manual submodule update.

aeslaughter committed:
Add moose paper bib file to allow use on website

(refs #14767)

permcody commented on modules/doc/content/help/inl/hpc_binary.md:
```diff
@@ -0,0 +1,251 @@
+1. `-l select=1:ncpus=48:mpiprocs=48` indicates to utilize one compute (`select=1`), utilizing all
+   48 cores and run with 48 mpi processes;
+1. `-l walltime=5:00` allocates 5 minutes of time for the simulation; and
+1. `-P moose` dictates that the job should part of the MOOSE queue.
```
I don't really have much time to go through this yet. I'll find that soon. One comment is that we should probably provide a list of projects or point them to a resource that has it. I think Eric/Tami like when people "charge" their time to the correct projects. There are projects for NEAMS, CASL, etc and we want to make sure hours go to the right places.

aeslaughter replied:
There is a list on HPC pages that I can link to, I will add it after you go through the rest. I am sure you will have more to say.

permcody commented on modules/doc/content/help/inl/hpc_binary.md:
````diff
@@ -0,0 +1,251 @@
+project queue.
+
+```bash
+qsub -I -l select=1:ncpus=1 -l walltime=1:0:0 -P moose
````
```suggestion
qsub -I -l select=1:ncpus=1 -l walltime=1:00:00 -P moose
```

permcody replied:
I would use a standard time format for instructional use. I actually checked your syntax to make sure it worked because it looked so funny to me.

permcody commented on modules/doc/content/help/inl/hpc_binary.md:
````diff
@@ -0,0 +1,251 @@
+
+```bash
+/apps/herd/bison/bison-opt -i input.i
+```
````
It might be worth pointing out to users that they won't be able to navigate to this folder somewhere. They will get a permission denied error and some might give up at that point.

permcody commented on modules/doc/content/help/inl/hpc_binary.md:
```diff
@@ -0,0 +1,251 @@
+refer to the [hpcweb.hpc.inl.gov](https://hpcweb.hpc.inl.gov/home/) for a list of the available
+systems.
+
+The second portion provides the commands to be executed. The `cd $PBS_O_WORKDIR` changes to the
```
👍

permcody commented on modules/doc/content/help/inl/hpc_remote.md:
```diff
@@ -0,0 +1,110 @@
+# Remote Access to INL-HPC
```
Odd that this file didn't get tracked as a move. Did you do `git mv`? We have a lot of history in this file. We will lose that here. Not a big deal, just odd.

aeslaughter replied:
I didn't pay attention, but I usually use `git mv`

aeslaughter replied:
I will see if I can fix it.

permcody reviewed (changes requested):
This looks good.

I think we should definitely link to the projects so people can fill in the appropriate project when following these instructions. Make sure you also link to some more complete documentation somewhere. Since this will be public, maybe just the PBS Professional manual or something. We could put a link there that works only for INL users, but I think that's tacky. 

There is another common option that you can add to this file to email you when your job completes (schedules, etc). Should we talk about that one too?

aeslaughter committed:
Improve INL-HPC documentation

(refs #14767)

aeslaughter commented:
@permcody This needs to merged with the broken apps, I will update them when this is merged.

permcody commented:
Approved
